store/src/java/com/zimbra/cs/filter/jsieve/EditHeaderExtension.java

```diff
@@ -299,11 +300,15 @@ public void setupEditHeaderData(Arguments arguments, AbstractCommand ac) throws
                         arg = itr.next();
                         if (arg instanceof StringListArgument) {
                             StringListArgument sla = (StringListArgument) arg;
-                            String newName = sla.getList().get(0);
-                            if (StringUtil.isNullOrEmpty(newName)) {
+                            String origNewName = sla.getList().get(0);
```

Comment:
are  you sure to get index 0 from the list? Potential array index out of bound case here. 
in the next line there is check for empty or null string, it indicates there is possibility of empty list. 
Can you please confirm ?